src/Components/test/E2ETest/Tests/ImageTest.cs

```diff
@@ -0,0 +1,355 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using BasicTestApp;
+using BasicTestApp.ImageTest;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests;
+
+public class ImageTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
+{
+    public ImageTest(
+        BrowserFixture browserFixture,
+        ToggleExecutionModeServerFixture<Program> serverFixture,
+        ITestOutputHelper output)
+        : base(browserFixture, serverFixture, output)
+    {
+    }
+
+    protected override void InitializeAsyncCore()
+    {
+        Navigate(ServerPathBase);
+        Browser.MountTestComponent<ImageTestComponent>();
+    }
+
+    private void ClearImageCache()
```

Comment:
Can we use `Brower.Manage().Cookies.DeleteCookieNamed('blazor-image-cache');`?

Reply:
This is not clearing a cookie, is it? It's clearing a Cache object from the caches API

Reply:
I don't think so, the docs say "Deletes the cookie with the specified name from the page." [It never mentions cache](https://www.seleniums.com/automation/notes/selenium/handling-cookies/manage-deletecookienamedstring-name/).